requirements.txt

```diff
@@ -1,6 +1,6 @@
 canonicalwebteam.flask_base==1.1.0
 canonicalwebteam.blog==6.4.0
-canonicalwebteam.discourse-docs==1.0.1
+canonicalwebteam.discourse==5.0.4
```

Comment:
latest version is `5.2.4`

Reply:
thanks - don't think we need that version

Reply:
We fix bugs and add features in later versions. But it's ok, it will get a renovate update anyway in the near future